datadog/provider.go

```diff
@@ -160,6 +160,7 @@ func Provider() *schema.Provider {
 			"datadog_monitor_json":                         resourceDatadogMonitorJSON(),
 			"datadog_organization_settings":                resourceDatadogOrganizationSettings(),
 			"datadog_role":                                 resourceDatadogRole(),
+			"datadog_saml_role_mapping":                    resourceDatadogRoleMapping(),
```

Comment:
Can you modify the name to authn_mappings because it corresponds to the [API name](https://github.com/DataDog/dogweb/blob/prod/dogweb/config/routing/authn_mapping.py)

Reply:
also leaves the door open for saml-to-other-thing mappings like saml-to-team :)